integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.4.19
+0.4.20
```

Comment:
Don't we need to at least bump `integrations/docker/images/chip-build-esp32/version` as well?  If not all the versions under `integrations/docker/images/`.

Reply:
Hi @bzbarsky-apple  Looks like it is a symlink. So, no changes required?

```
$ ls -l integrations/docker/images/chip-build-esp32/version
lrwxrwxrwx 1 hrishi hrishi 21 Dec 11 00:12 integrations/docker/images/chip-build-esp32/version -> ../chip-build/version
```

Reply:
Ah, excellent.